pwa/src/layout/Layout.tsx

```diff
@@ -41,13 +40,15 @@ const Layout: React.FC<LayoutProps> = ({ children, pageContext, location }) => {
       <GlobalProvider value={[globalContext, setGlobalContext]}>
         <Head />
         <APIProvider value={API}>
-          <Document className={clsx(process.env.GATSBY_NL_DESIGN_THEME_CLASSNAME, styles.document)}>
-            <Toaster position="bottom-right" />
-
-            <div className={styles.container}>
-              <Content {...{ children }} />
-            </div>
-          </Document>
+          <Surface>
+            <Document className={process.env.GATSBY_NL_DESIGN_THEME_CLASSNAME}>
```

Comment:
I see that you've added the className in `Head`. It's also here, I'd expect us to only need one.